lighthouse-core/report/html/report-styles.css

```diff
@@ -113,11 +110,21 @@
   --score-title-line-height-big: 36px;
   --score-title-line-height: 26px;
   --section-padding: 40px;
+  --shape-margin: 7px 12px 0 4px;
+  --shape-size: 12px;
```

Comment:
this would probably be a little more helpful with an additional qualifier in the variable name. Which shape? (same with `circle` and `square`?)

Reply:
it's used for both metric icons and audit score icons so I went with `--icon-shape-size`...

Reply:
> it's used for both metric icons and audit score icons so I went with `--icon-shape-size`...

"icon" and "shape" mean basically the same thing :)

what about `--score-shape-size`, etc?